src/containers/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,90 @@
+import {
+  AppBar,
+  CssBaseline,
+  IconButton,
+  Divider,
+  Drawer,
+  Toolbar,
+  List, Box
+} from '@material-ui/core';
+
+import MenuIcon from '@material-ui/icons/Menu';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import AssignmentIcon from '@material-ui/icons/Assignment';
+import LoopIcon from '@material-ui/icons/Loop';
+import ExitToAppIcon from '@material-ui/icons/ExitToApp';
+
+import clsx from 'clsx';
+import { FC, useState } from 'react';
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { logOut } from '../../store/auth/actions';
+import { CmwlListItemLink } from '../../components/UI/CmwlListItemLink/CmwlListItemLink';
+import {
+  DASHBOARD_CMWL_TITLE,
+  DASHBOARD_EXECUTIONS_NAV_PRIMARY,
+  DASHBOARD_PROJECTS_NAV_PRIMARY
+} from '../../shared/Constants';
+import { CmwlTypographyLink } from '../../components/UI/CmwlTypographyLink';
+import { useDashboardStyles } from './DashboardStyles';
+
+export const Dashboard: FC = props => {
+  const dispatch = useDispatch();
+  const classes = useDashboardStyles();
+  const [open, setOpen] = useState(false);
+  const handleDrawerOpen = () => {
+    setOpen(true);
+  };
+  const handleDrawerClose = () => {
+    setOpen(false);
+  };
+
+  const logOutHandler = () => {
+    dispatch(logOut());
+  };
+
+  return (
+    <Box className={classes.root}>
+      <CssBaseline />
+      <AppBar position='absolute' className={clsx(classes.appBar, open && classes.appBarShift)}>
```

Comment:
Most (if not everyone) of MUI components expose `classes` field. It accepts a map from class name to boolean, so you can omit using clsx and use `classes` instead (and I recommend doing it):

```ts
classes={{
  [classes.appBar]: true,
  [classes.appBarShift]: open
}}
```

Reply:
Reference: https://material-ui.com/customization/components/#overriding-styles-with-classes

Reply:
This looks great!

Could you please help to get where my mistake is cause it doesn't work in my case?

The documentation about classes talks that classes using for overriding and extending styles.
As I understand the expected map is a map from class name to string.
`export type ClassNameMap<ClassKey extends string = string> = Record<ClassKey, string>;`

When I pass a map from class name to boolean I don't receive any warnings but as a result, the styles are not applied.

Also, I've tried to find examples but unfortunately without success.

Thank you in advance!

Reply:
Sure, here's an example:

```ts
import { createStyles, makeStyles } from '@material-ui/core';

const useStyles = makeStyles(theme => createStyles({
  cmwlButton: {
    backgroundColor: theme.palette.secondary.contrastText,
    borderRadius: 4,
    '&:disabled': {
      backgroundColor: blueGrey['50'],
    },
  },
}));

const CmwlButton: React.FC<CmwlButtonProps> = ({ /* ... */ }) => {
  const classes = useStyles();
  const cmwlButtonClasses = { root: classes.smsButton };

  return (
    <Button classes={cmwlButtonClasses} />
  );
};
```

Reply:
In order to use a map from class name to boolean, I've added `classnames`